content/en/ddsql_reference/ddsql_preview/reference_tables.md

```diff
@@ -1,11 +1,24 @@
 ---
 title: Reference Tables in DDSQL
+aliases:
+- /ddsql_editor/reference_tables/
 further_reading:
 - link: "/integrations/guide/reference-tables/"
   tag: "Documentation"
   text: "Add Custom Metadata with Reference Tables"
+- link: "/ddsql_reference/"
+  tag: "Documentation"
+  text: "DDSQL Reference"
 ---
 
+{{< callout url="https://datadoghq.com/private-beta/ddsql-editor">}}
+The DDSQL Editor is in Preview.
+{{< /callout >}}
+
+<div class="alert alert-warning">
+  There are two different <strong>variants</strong> of DDSQL. See the syntax documented in <a href="/ddsql_reference/">DDSQL Reference</a>.
```

Comment:
I'm a little confused about which version these guides are meant for. The URL seems to indicate the DDSQL Editor, but it's not in the topic titles, and this warning says to see the syntax for the current version, but doesn't tell me why. Can we make this clearer?

Reply:
Thanks Janine, the guide is using the Preview syntax so that's why it's under the Preview section. And right now, the only product that uses the preview syntax is the Editor, but there's going to be period where it uses both and then deprecates the preview syntax. At which point, we remove the Preview docs.

I'll remove mention of the Editor in the Preview docs since that adds to the confusion.

Reply:
Right! That's definitely an awkward stage in between. I think what confused me was that the topic titles were different, so some of them clearly say `Editor` and `(Preview)`, but some of them don't. Is that on purpose?

Reply:
Nope not intentional, I removed for all of them, hopefully it's less confusing! I removed it from the main tabs and forgot about the child pages 😄

Reply:
Thanks! No worries 🙂 Is it intentional that the pages under `DDSQL (Preview) References` have `(Preview)` in their titles, but the pages under `Guides for using DDSQL (Preview) Queries` don't?

Reply:
No it's not intentional, but they're more use case guides while the other pages are actively informing on syntax which is in Preview. I can add the Preview to these pages to the guides as well.

Reply:
Ahh okay. That makes sense! I think I was getting tripped up about what on that page was in preview or not. Maybe adding `(Preview)` could help set expectations for users who are/aren't currently using the DDSQL Editor?

Reply:
I'm still not seeing `(Preview)` in the topic title for `/ddsql_reference/ddsql_preview/ddsql_use_cases/` and am not sure if that's on purpose!

I'm also not sure why the build check is failing - maybe updating your branch from `master` might fix that?

Reply:
Thanks for catching this, I missed it!